client/post-editor/editor-diff-viewer/changes.jsx

```diff
@@ -0,0 +1,40 @@
+/**
+ * External dependencies
+ *
+ * @format
```

Comment:
I think this declaration has to go in its own comment block to appease the linter